src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const CAR_COST = 40;
+  const SHORT_TERM_DISCOUNT = 20;
+  const LONG_TERM_DISCOUNT = 50;
+  const PRICE = days * CAR_COST;
+
+  if (days >= LONG_TERM) {
+    return PRICE - LONG_TERM_DISCOUNT;
+  }
+
+  if (days < SHORT_TERM) {
+    return PRICE;
```

Comment:
The condition for rentals less than SHORT_TERM is correct, as it returns the full price without any discount.